### ISSUE TITLE
Delete bookmark

### ISSUE THREAD

**Issue by aristidesfl** [gillibrand/launchbar-pinboard#4]
Is there a way to delete a bookmark?

Thanks

**Comment by gillibrand** [comment]
No. I wouldn't think that would be the kind of action you need quick access to too often. Maybe it could be done with a modifier key in the existing actions.

I haven't looked at this in a while, so no promises, but I can take a look at adding that sometime.

**Comment by aristidesfl** [comment]
That would be awesome :)
Don't know why but I find myself in need to delete old bookmarks
